hack/Makefile

```diff
@@ -14,8 +14,8 @@ generate-python-client:
 	./hack/generate.sh generate_python_client
 
 generate-mocks:
-	find . -name 'mock_*.go' -type f -delete
-	go generate -v $(shell go list ./... | grep -v 'assisted-service/models\|assisted-service/client\|assisted-service/restapi')
+	find . -name 'mock_*.go' -type f  -not -path './internal/*' -and -not -path './mocks/*' -and -not -path './pkg/*' -and -not -path './vendor/*' -delete
+	go generate -run="--build_flags=--mod=mod" -v $(shell go list ./... | grep -v 'assisted-service/models\|assisted-service/client\|assisted-service/restapi')
```

Comment:
wait a second
I think ``go generate -run`` isn't really the way to pass those ``build_flags`` arguments. It allows providing a regex for filtering the ``go:generate`` directives
If not possible globally, then we should revert back to having the build_flags for each ``go:generate`` directive

Reply:
I don't think there is any other alternative. I am going back to the original changes.